### PR TITLE
learning-goals-readme-done #2

### LINKED ISSUE #2: learning goals
- [ ] This is issue is for the readme of the Learning Goals

### PR DESCRIPTION
**Pull request by aandom**
uploaded learning goals , resolves #2

### CODE REVIEW

File: collaboration/learning_goals.md [hunk 1]
```diff
@@ -1,5 +1,81 @@
...
+### *Alexander Andom*
+
+- *Learn New Data Structures*: Learn and implement new data structures that are
+unfamiliar or challenging.
+- *Improve Debugging Skills*: Develop strategies for debugging code and
+resolving issues more efficiently.
+- *Deepen Knowledge of Testing Frameworks*: Learn to use `unittest` in Python
+to write solid and detailed unit tests.
+
+### *Mohamed Saed*
```
Comment (Tbaosman) [inline collaboration/learning_goals.md:72]: would you mind if you change Saed to Saeed

Comment (Tbaosman) [inline collaboration/learning_goals.md]: Can you please change Saed to Saeed

**Comment by aandom** [comment]
thank you saeed for the comment, I just fixed the typo error.

**Review by Tbaosman — APPROVED** [review]

**Comment by Tbaosman** [comment]
Nice work!

**Pull request merged**